src/java/org/apache/nutch/parse/ParseUtil.java

```diff
@@ -237,7 +237,7 @@ public void processSitemapParse(String url, WebPage page,
           try {
             reversedUrl = TableUtil.reverseUrl(toUrl); // collect it
           } catch (MalformedURLException e) {
-            e.printStackTrace();
+            LOG.error("MalformedURLException occurred: ", e);
```

Comment:
That a specific exception happened is obvious, would be better to explain what happened with which input, e.g.
```
LOG.error("Failed to reverse URL {}: {}", toUrl, e.getMessage());
```
This also applies to many other of the new log messages.